babel.config.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
     '@babel/preset-typescript',
     ['@babel/preset-env', { targets: { node: 10 } }],
   ],
+  ignore: ['src/**/__tests__/**'],
```

Comment:
This change means we're releasing ~1 second faster, since we're now only compiling ~48 files instead of ~90 😂